content/en/docs/instrumentation/erlang/testing.md

```diff
@@ -0,0 +1,132 @@
+---
+title: "Testing"
+weight: 40
+---
+
+When relying on OpenTelemetry for alerts, pages, or SLIs/SLOs, it can be important to test that certain spans are created and attributes correctly set.
```

Comment:
```suggestion
When relying on OpenTelemetry for your Observability needs, it can be important to test that certain spans are created and attributes correctly set. For example, can you be sure that you attaching the right metadata to data that ultimately powers an SLO? This document covers an approach to that kind of validation.
```

Reply:
Ah this is great.